src/nimdowpkg/wmcommands.nim

```diff
@@ -27,6 +27,8 @@ type
     wmcDestroySelectedWindow = "destroyselectedwindow",
     wmcToggleFloating = "togglefloating",
     wmcJumpToUrgentWindow = "jumptourgentwindow"
+    wmcIncWidthDiff = "incwidthdiff"
+    wmcDecWidthDiff = "decwidthdiff"
```

Comment:
I think we should come up with a name that better indicates what the config property is, but I haven't thought of one yet